python/tvm/relay/op/_transform.py

```diff
@@ -50,11 +51,13 @@
 _reg.register_schedule("strided_slice", schedule_injective)
 _reg.register_schedule("slice_like", schedule_injective)
 _reg.register_schedule("split", schedule_injective)
+_reg.register_dynamic_compute("split", True)
```

Comment:
Can ops automatically register dynamic compute? It seems like this would be a useful feature for every op.

Reply:
we can make it by default true, because the false case will break immediately, then we can add annote.

Reply:
Personally, I think that's a little cleaner, but we can wait to see what other reviewers think :)